extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -159,6 +160,17 @@ export const RANDOM_SUBDOMAIN_SAFEFRAME_BRANCHES = {
   EXPERIMENT: '21065818',
 };
 
+/**
+ * @const @enum {string}
+ * @visibleForTesting
+ * TODO(#28555): Clean up experiment
+ */
+export const EXPAND_JSON_TARGETING_EXP = {
+  ID: 'expand-json-targeting',
+  CONTROL: '21066261',
+  EXPERIMENT: '21066262',
+};
+
```

Comment:
This is a tricky thing to run an experiment on, since presumably no one will be trying to use it yet.

Reply:
The idea was that this call to `expandStringAsync` might introduce some latency even if there are no macros to be expanded. If you think it is not worth it to run the experiment I am happy to remove. WDYT?

Reply:
Ah, thanks, that's a good point! I do think it would be nice to run the experiment, then.